source/_data/products.yml

```
@@ -0,0 +1,449 @@
+      url: https://shop.swiss-domotique.ch/de/sound/2667-nabu-casa-home-assistant-voice-preview-edition.html
+      logo: /images/distributors/swiss-domotique.webp
+    - name: The Pi Hut
+      ship_from: UK
+      ship_to: Europe
+      url: https://thepihut.com/products/home-assistant-voice-preview-edition
+      logo: /images/distributors/pi-hut.webp
+    - name: Everything Smart Technology
+      ship_from: UK
+      ship_to: Europe
+      url: https://shop.everythingsmart.io/products/home-assistant-voice-preview-edition
+      logo: /images/distributors/est.webp
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)